test/controllers/api/v2/organizations_controller_test.rb

```diff
@@ -89,6 +89,95 @@ def test_create_duplicate_name
       assert_response :unprocessable_entity
     end
 
+    def test_create_with_auto_label
+      Organization.any_instance.stubs(:redhat_repository_url)
+      Organization.any_instance.stubs(:default_content_view).returns(OpenStruct.new(id: 1))
+      Organization.any_instance.stubs(:library).returns(OpenStruct.new(id: 10))
+      name = "Organization With Label"
+      assert_sync_task ::Actions::Katello::Organization::Create do |org|
+        org.name.must_equal name
+        assert org.valid?
+        org.label.wont_be_empty
```

Comment:
it would be better to check that the label is a certain value, as the labels should follow known rules. in this case, shouldn't it be
```"organization_with_label"```

Reply:
@JacobCallahan many thanks for your review.

The purpose of this  test is not to check the correctness of label value, but that label value has been assigned when organization was created without it.
Note: robottelo use the same principe for this test.

Think that a more general stand alone unit-test function should be written for function "Util::Model.labelize", but this is not in the scope of this test

Reply:
It seems then that I just disagree with the value of this assertion over a true value check. If everyone else is fine with it, I won't block.